setup.cfg

```diff
@@ -31,6 +31,7 @@ packages = find:
 setup_requires = incremental
 python_requires = >=3.6
 install_requires =
+    docutils
```

Comment:
Users that only need epytext support and don't build the docs don't need `docutils` installed. So I think it would be better to have it as an extra.